presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -45,6 +47,39 @@ public Estimate getDistinctValuesCount()
         return distinctValuesCount;
     }
 
+    @Override
```

Comment:
Please extract separate commit with adding toString/hashCode/equals to Stats SPI.